org.libreoffice.LibreOffice.json

```diff
@@ -0,0 +1,399 @@
+                    "dest-filename": "external/tarballs/openldap-2.4.44.tgz"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/liborcus-0.12.1.tar.gz",
+                    "sha256": "676b1fedd721f64489650f5e76d7f98b750439914d87cae505b8163d08447908",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/liborcus-0.12.1.tar.gz"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/poppler-0.56.0.tar.xz",
+                    "sha256": "869dbadf99ed882e776acbdbc06689d8a81872a2963440b1e8516cd7a2577173",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/poppler-0.56.0.tar.xz"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/c0b4799ea9850eae3ead14f0a60e9418-postgresql-9.2.1.tar.bz2",
+                    "sha256": "db61d498105a7d5fe46185e67ac830c878cdd7dc1f82a87f06b842217924c461",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/c0b4799ea9850eae3ead14f0a60e9418-postgresql-9.2.1.tar.bz2"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/a39f6c07ddb20d7dd2ff1f95fa21e2cd-raptor2-2.0.15.tar.gz",
+                    "sha256": "ada7f0ba54787b33485d090d3d2680533520cd4426d2f7fb4782dd4a6a1480ed",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/a39f6c07ddb20d7dd2ff1f95fa21e2cd-raptor2-2.0.15.tar.gz"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/1f5def51ca0026cd192958ef07228b52-rasqal-0.9.33.tar.gz",
+                    "sha256": "6924c9ac6570bd241a9669f83b467c728a322470bf34f4b2da4f69492ccfd97c",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/1f5def51ca0026cd192958ef07228b52-rasqal-0.9.33.tar.gz"
+                },
+                {
+                    "url": "https://dev-www.libreoffice.org/src/e5be03eda13ef68aabab6e42aa67715e-redland-1.0.17.tar.gz",
+                    "sha256": "de1847f7b59021c16bdc72abb4d8e2d9187cd6124d69156f3326dd34ee043681",
+                    "type": "file",
+                    "dest-filename": "external/tarballs/e5be03eda13ef68aabab6e42aa67715e-redland-1.0.17.tar.gz"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "./autogen.sh --prefix=/run/build/libreoffice/inst --with-distro=LibreOfficeFlatpak",
+                "make",
+                "make distro-pack-install-strip",
+                "make cmd cmd='$(SRCDIR)/solenv/bin/assemble-flatpak.sh'"
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=system-bus",
```

Comment:
Does it really require access to entire session and system bus?

Reply:
To be honest, I haven't looked at these settings in quite a while.  I set them up in the early days of building LO flatpaks, and that's what we use ever since for <http://www.libreoffice.org/download/flatpak/>.

Reply:
I will play with it and see what exactly is needed.

Reply:
`system-bus` is definitely not needed. It fails to start without `session-bus`, but I don't know which d-bus name it tries to claim or communicate with.

Reply:
It uses "org.libreoffice.LibreOfficeIpc0" for communication when multiple instances of LO are started, see <https://opengrok.libreoffice.org/xref/core/desktop/source/app/officeipcthread.cxx#454>.  (There must only be one instance of LO running per user at any time, so additional instances pass their command line args to the first instance and quit again.  This had traditionally been implemented with named pipes; I added the DBus variant specifically for Flatpak.)

Reply:
Thanks, added commits to own that name instead of accesing the bus directly.